Engine/src/Engine/ResourceManagement/ShaderGenerator.cpp

```diff
@@ -60,4 +55,50 @@ namespace Engine
 		}
 		return true;
 	}
+
+	ShaderNorms::ShaderNorms() : m_defaultShader("TextureFill")
```

Comment:
would it be possible to have an enum for these shaders instead of a string map?

Reply:
Yes!